x-pack/plugins/case/server/client/types.ts

```diff
@@ -25,43 +27,34 @@ import {
   AlertServiceContract,
 } from '../services';
 import { ConnectorMappingsServiceSetup } from '../services/connector_mappings';
-import type { CasesRequestHandlerContext } from '../types';
 import { CaseClientGetAlertsResponse } from './alerts/types';
 
-export interface CaseClientCreate {
```

Comment:
More of a question. We seem to be passing in the post and patch request types while the others are defined here? Is there a reason they're separated?

Reply:
I think that might have been from more of a convenience thing where some parameter types were defined already by the io-ts types and we just pass those fields around where as others don't strictly align with an io-ts type.